File: data/src/main/java/com/everis/data/model/LocalContactEntityDataMapper.java
package com.everis.data.model;

import com.everis.domain.model.LocalContact;

/**
 * Created by everis on 3/05/18.
 */

public class LocalContactEntityDataMapper {

    public static LocalContact transform(LocalContactEntity localContactEntity){
        LocalContact localContact = new LocalContact();
        localContact.setName(localContactEntity.getName());
        localContact.setNumber(localContactEntity.getNumber());
        return localContact;
    }

}

File: data/src/main/java/com/everis/data/model/LocalContactEntityDataMapper.kt
package com.everis.data.model

import com.everis.domain.model.LocalContact

object LocalContactEntityDataMapper {

    fun transform(localContactEntity: LocalContactEntity): LocalContact {
        val localContact = LocalContact()
        localContact.name = localContactEntity.name
        localContact.number = localContactEntity.number!!.replace("[^0-9]".toRegex(), "")
        return localContact
    }

}
